index.html

```diff
@@ -1 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="nav-bar">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav>
+            <button type="button" class="hamburger">&#9776;</button>
+        </nav>
+    </header>
+    <main>
+        <section class="main-content">
+            <div class="content">
+                <h1>I'm Rabecca Glad to see you!</h1>
+                <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                <a href="#" class="social">LET'S CONNECT</a><br>
+               <img src="images/social-icons.png" class="social-media-icons" alt="social-media-icons">
```

Comment:
- [ ] Please remove this image and use social media links with anchor tags. Also, remember to wrap your links in a list, When we have similar elements It is a good practice to wrap them in a list.